SampleGallery/Shared/SampleDefinition.cs

```diff
@@ -160,6 +160,7 @@ static public void RefreshSampleList()
                 new SampleDefinition(SwipeScroller.StaticSampleName,                typeof(SwipeScroller),                SampleType.EndToEnd,  SampleCategory.DynamicHumanInteractions,   false, false, dateAdded: new DateTime(2017,03,05),  imageUrl: "ms-appx:///Assets/SampleThumbnails/SwipeScroller.PNG",           sdkVersion: RuntimeSupportedSDKs.SDKVERSION._14393),
                 new SampleDefinition(ShyHeader.StaticSampleName,                    typeof(ShyHeader),                    SampleType.EndToEnd,  SampleCategory.DynamicHumanInteractions,   false, false, dateAdded: new DateTime(2017,04,25),  imageUrl: "ms-appx:///Assets/SampleThumbnails/ShyHeader.PNG",               sdkVersion: RuntimeSupportedSDKs.SDKVERSION._14393, featured: true),
                 new SampleDefinition(BlurPlayground.StaticSampleName,               typeof(BlurPlayground),               SampleType.Reference, SampleCategory.Conceptual,                 true,  true,  "ms-appx:///Assets/SampleThumbnails/BlurPlayground.PNG",                                                          sdkVersion: RuntimeSupportedSDKs.SDKVERSION._14393),
+                new SampleDefinition(Win2DInterop.StaticSampleName,                 typeof(Win2DInterop),                 SampleType.Reference, SampleCategory.Conceptual,                 true,  true,  "ms-appx:///Assets/SampleThumbnails/Win2DInterop.PNG",                  sdkVersion: RuntimeSupportedSDKs.SDKVERSION._14393 ),
```

Comment:
Please sync to the latest samplegallery and merge with the latest changes.  There is a change required here for SampleType and a missing parameter.